dfaligner/model.py

```diff
@@ -127,6 +128,11 @@ def check_and_upgrade_checkpoint(self, checkpoint):
             # Upgrading from 0.0 to 1.0 requires no changes; future versions might require changes
             checkpoint["model_info"]["version"] = "1.0"
 
+        # We changed the handling of phonological features in everyvoice==0.3.0
+        if ckpt_version < Version("1.1"):
+            raise ValueError(f"""There were breaking changes to the handling of text in version 1.1, introduced in version 0.3.0 of EveryVoice.
```

Comment:
Just "version 1.1" like this is confusing, because various things have versions in this message. Name it: "in dfagliner model version 1.1" or whatever strings accurately describes this.